QUEST_LV_0200.tsv

```diff
@@ -5591,57 +5591,57 @@ QUEST_LV_0200_20150323_005590	Upham was seperated from the colleagues due to the
 QUEST_LV_0200_20150323_005591	You've took Upham to Broknews. Talk with Broknews.
 QUEST_LV_0200_20150401_005592	How would we know about the mind of the baron.{nl}He told us to behave well when the revelator comes so we are just doing the work he told us to do.{nl}
 QUEST_LV_0200_20150401_005593	Well, if you see the baron yourself, you may find out something.{nl}He is working on something big at Mirukiti farm, so why don't you go meet him?
-QUEST_LV_0200_20150401_005594	I have to tie the sacks of the grains, but I don't have enough strings.{nl}I am sure that the monsters near here took them. Please prepare tight strings for me.
+QUEST_LV_0200_20150401_005594	$I have to tie the sacks of the grains, but I don't have enough rope.{nl}I am sure that the monsters near here took them. Please prepare tight rope for me.
```

Comment:
I have to tie these sacks of grain, but I don't have enough rope.{nl}'m sure that the monsters near here took them. Please get some tight rope for me.